tools/ca/GeoTrustGlobalCA.pem

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
![The Scream](http://4.bp.blogspot.com/-KZwCe43wOBc/TeKEDqbrBmI/AAAAAAAAApM/psvtMV-ibww/s640/edvard-munch-the-scream.jpg)